pkg/skaffold/deploy/kubectl.go

```diff
@@ -31,6 +36,35 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Slightly modified from kubectl run --dry-run
+var deploymentTemplate = template.Must(template.New("deployment").Parse(`apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
you need a service too right?

Reply:
You don't *need* one, only if you want to access it over an LB or something. I think we should probably recommend port-forward in any samples.